### DISCUSSION TITLE
Paperless NGX no Email Consumption

### DISCUSSION THREAD

**Discussion by Tolbag** [tteck/Proxmox#3517] [Help Needed] [answered]
Hey,

I have set up and tested an email account. The connection should be working. I have also set up a rule.

However, the tab "mail.log" is not displayed in the LOG.

Where could be the problem?

Paperless Version is 2.11.1 and 2.11.4

Regards

![image](https://github.com/user-attachments/assets/a61df959-b635-44e9-9647-ce368fae1145)
![image](https://github.com/user-attachments/assets/7777c9a8-5dfc-4da7-8b98-ccce593e9569)
![image](https://github.com/user-attachments/assets/be767f69-03ab-48b8-8232-2a38d17daa9e)

**Comment by smignon612** [comment]
same problem here, it worked untill 2 weeks ago.
when i use the test the connection, is it ok.

**Comment by Mellowlynx** [comment, marked as answer]
I had this to about a month ago, this fixed it for me:
https://github.com/paperless-ngx/paperless-ngx/discussions/3350#discussioncomment-5842310

Check if the `paperless-scheduler.service` indeed fails, rename the .db file and restart service.

**Reply by Tolbag** [reply]
I think this fixed it for me too.
Thank you!

**Reply by smignon612** [reply]
where is that db located?

**Reply by Tolbag** [reply]
/opt/paperless/data

**Reply by smignon612** [reply]
working here now to

**Reply by hundehausen** [reply]
```shell
mv /opt/paperless/data/celerybeat-schedule.db.db /opt/paperless/data/celerybeat-schedule.db.db.bak
systemctl restart paperless-scheduler.service
```
